client/src/App.js

```diff
@@ -22,15 +27,26 @@ export default function App () {
     getMovies();
   }, []);
 
-  const addToSavedList = id => {
+  //Getting an error that below is not a function - not sure if it's from below, or from the props and how I'm trying to use it in the Movie.js
+  const addToSavedList = movie => {
     // This is stretch. Prevent the same movie from being "saved" more than once
-  };
+    if (movie.indexOf(movie.title) != -1) {
```

Comment:
Also since movie is the object being passed here the name of the array you're checking it's a part of also called movie? If it is one of these names needs to change in order for this to work correctly.